### ISSUE TITLE
Alabama 2000 general presidential results are doubled

### ISSUE THREAD

**Issue by GlutenFreeGrapes** [fivethirtyeight/election-results#2]
I looked through the election_results_presidential.csv file and saw the data for the Alabama 2000 general presidential election. It seemed suspicious that not only had the entries for all candidates been duplicated (with the exception of the `id` column), the percentages seemed oddly small (it seems unlikely that George W. Bush won Alabama with a mere 28% of the vote and a margin of victory of around 7.5%)
![issue](https://user-images.githubusercontent.com/70921630/211130847-db16c8f1-34bc-47c7-a222-a0fdc7de4719.PNG)

Using the source provided in the `url` column ([here](https://history.house.gov/Institution/Election-Statistics/)), I looked at the section on Alabama's results, and sure enough: 
![otherissue](https://user-images.githubusercontent.com/70921630/211130892-f8801299-4614-4dee-a00f-cf114f238917.PNG)
Summing up the votes in the `votes` column for this election would have landed on a figure of 3,332,544, twice the official statistic of 1,666,272.
